.gitignore

```diff
@@ -28,3 +28,4 @@
 !/app/assets/builds/.keep
 
 /node_modules
+/Gemfile
```

Comment:
Why we need to ignore `Gemfile` ?

Reply:
We need both Gemfile and Gemfile.lock
I checked if it came from a branch before, but not.
My mistake, sorry.